extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -130,13 +133,11 @@ static public function latestFromTitle( Title $title, array $param = [] ) {
 	}
 
 	/**
+	 * Used to store extra data in comment content
 	 *
-	 * Used to store extra data in comment contend
-	 *
-	 * @access public
```

Comment:
Removed all these `@access` and `@static` annotations.  They add nothing over the PHP keywords used in the method signature.